docs/container_registry.md

```diff
@@ -103,103 +120,17 @@ Then update the `volumes` and `environment` sections of your `gitlab` container:
             - ./certs:/certs
 ```
 
-### Nginx Site Configuration
-
-```nginx
-server {
-    root /dev/null;
-    server_name registry.example.com;
-    charset UTF-8;
-    access_log /var/log/nginx/registry.example.com.access.log;
-    error_log /var/log/nginx/registry.example.com.error.log;
-
-    # Set up SSL only connections:
-    listen *:443 ssl http2;
-    ssl_certificate             /etc/letsencrypt/live/registry.example.com/fullchain.pem;
-    ssl_certificate_key         /etc/letsencrypt/live/registry.example.com/privkey.pem;
-
-    ssl_ciphers 'ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256:ECDHE-RSA-AES256-SHA:ECDHE-RSA-AES128-SHA:ECDHE-RSA-DES-CBC3-SHA:AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-SHA256:AES128-SHA256:AES256-SHA:AES128-SHA:DES-CBC3-SHA:!aNULL:!eNULL:!EXPORT:!DES:!MD5:!PSK:!RC4';
-    ssl_protocols  TLSv1 TLSv1.1 TLSv1.2;
-    ssl_prefer_server_ciphers on;
-    ssl_session_cache  builtin:1000  shared:SSL:10m;
-    ssl_session_timeout  5m;
-
-    client_max_body_size        0;
-    chunked_transfer_encoding   on;
-
-    location / {
-        proxy_set_header  Host              $http_host;   # required for docker client's sake
-        proxy_set_header  X-Real-IP         $remote_addr; # pass on real client's IP
-        proxy_set_header  X-Forwarded-For   $proxy_add_x_forwarded_for;
-        proxy_set_header  X-Forwarded-Proto $scheme;
-        proxy_read_timeout                  900;
-        proxy_pass        http://localhost:5000;
-    }
-}
-
-server {
-    listen *:80;
-    server_name  registry.example.com;
-    server_tokens off; ## Don't show the nginx version number, a security best practice
-    return 301 https://$http_host:$request_uri;
-}
-```
```

Comment:
@drubin I think at least the crucial parts of the Nginx configuration should remain. E.g. which `proxy_set_header` you should have and the `proxy_pass`.

I've actually added the full config to leave no questions about how the setup works. So I'd say, we should keep all of it.

Reply:
The reason I removed them is the entire setup requires working proxy server setup and it. The assumption nd the docs don't match

> We assume that you already have Nginx installed on your host system

I would  rather add an additional doc on setting up a proxy for gitlab and the registry and link to that. What do you think?

Reply:
Not sure what you mean here:

> We assume that you already have Nginx installed on your system

Yes, Nginx is installed. And it may already be configured for the GitLab site (without the registry). But not the registry. That's why I included the configuration that you have to add to your existing Nginx installation.

Maybe we could change that sentence to:

> We assume that you already have Nginx installed and use it as reverse proxy for GitLab. We add another site configuration for the Container registry.